socorro/signature/rules.py

```diff
@@ -154,9 +154,10 @@ def normalize_rust_function(self, function, line):
 
     def normalize_cpp_function(self, function, line):
         """Normalizes a single cpp frame with a function"""
-        # If the function ends in "const", drop that
-        if function.endswith(' const'):
-            function = function[:-6]
+        # Drop member function cv/ref qualifiers like const, const&, &, and &&
+        for ref in ('const', 'const&', '&&', '&'):
```

Comment:
This covers things we've seen and what's here:

https://en.cppreference.com/w/cpp/language/member_functions

We can add others as they come up.